payload/builder.go

```diff
@@ -84,7 +84,7 @@ func (p *Payload) UnsetBadge() *Payload {
 // This will play a sound from the app bundle, or the default sound otherwise.
 //
 //	{"aps":{"sound":sound}}
-func (p *Payload) Sound(sound string) *Payload {
+func (p *Payload) Sound(sound interface{}) *Payload {
```

Comment:
This can still stay as interface, that way it can accept either a string, map or the sound struct.